src/file-upload/styles.scss

```diff
@@ -0,0 +1,84 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+@use '../internal/styles/tokens' as awsui;
+
+.root,
+.upload-button,
+.upload-input,
+.file-option-name-edit-activate,
+.file-option-name-edit-cancel,
+.file-option-name-edit-submit,
+.file-option-name-input,
+.file-option-type,
+.file-option-size,
+.file-option-last-modified,
+.file-option-thumbnail {
+  /* used in test-utils */
+}
+
+.file-option {
+  width: 100%;
+  min-width: 0;
+  display: flex;
+  gap: awsui.$space-scaled-xs;
+}
+
+.file-option-thumbnail {
+  max-width: 60px;
+}
+
+.file-option-thumbnail-image {
+  width: 100%;
+  height: auto;
+}
+
+.file-option-metadata {
+  width: 100%;
+  overflow: hidden;
+}
+
+.file-option-name-edit-activate {
+  flex-shrink: 0;
+}
+
+.file-option-name-edit-submit {
+  margin-right: 1px;
```

Comment:
Can we use a token like `awsui.$space-scaled-xs`?

Reply:
No longer applicable.